be/src/olap/rowset/segment_v2/inverted_index_fs_directory.cpp

```diff
@@ -82,7 +82,6 @@
 namespace doris::segment_v2 {
 
 const char* const DorisFSDirectory::WRITE_LOCK_FILE = "write.lock";
-const char* const DorisFSDirectory::COMPOUND_FILE_EXTENSION = ".idx";
 
 class DorisFSDirectory::FSIndexOutput : public lucene::store::BufferedIndexOutput {
```

Comment:
`DorisFSDirectory::FSIndexInput` is declared in `inverted_index_fs_directory.h`, while `DorisFSDirectory::FSIndexOutput` is declared in `inverted_index_fs_directory.cpp`.